src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -171,7 +179,10 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
               ></paper-icon-button>
             </div>
           </div>
-          <div class="title-controls">
+          <div
+            class="title-controls"
+            style="padding-right: ${this._cardWidth - 16}px"
```

Comment:
Doesn't this work?

```suggestion
            style="padding-right: calc(100% - 16px)"
```

Reply:
I suck at paying attention to what I am doing. Client height